exe/wallet/Main.hs

```diff
@@ -415,6 +415,9 @@ execHttpBridge args _ = do
     minLogSeverity <- getOptionValue <$>
         args `parseArg` longOption "min-log-severity"
     tracer <- initTracer minLogSeverity "serve"
+    logInfo tracer $ "Wallet backend server STARTED. "
+        <> "Version "
+        <> T.pack (showVersion version)
```

Comment:
Understood! I'll move this to the other location. :)

Reply:
In effect we already have a message to indicate that the server has **started**:
https://github.com/input-output-hk/cardano-wallet/blob/master/exe/wallet/Main.hs#L427-L428

Therefore, we probably don't need another similar message.

However, it **is** useful to output the **version string**. Therefore, I've changed this PR so that it prints "starting" rather than "started".